src/collections/_documentation/guides/grouping-and-fingerprints/index.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Grouping & Fingerprints
+sidebar_order: 0
+---
+
+Have you ever had a set of similar-looking issues like this?
+![Issues Dashboard]({% asset guides/grouping-and-fingerprints/issues-dashboard.png @path %})
+
+Let's first understand what makes events group into an issue. testTypeIssue15 is an issue that had 4 events. These events grouped into the issue because they shared the same *fingerprint*
+
+**What is a fingerprint?**
+A fingerprint is a way to uniquely identify an event. Sentry sets a fingerprint by default for you. Sentry uses its own built-in grouping algorithms to generate a fingerprint based on information available within the event such as a stack trace, exception, and message. Events with the same fingerprint are grouped together.
+
+**Do I need to do anything?** No. A stack trace is used by default if it's available. So if two events have the same stack trace, then they will group into one issue. If you have separate issues that you'd like to group together, then this guide will show you how.
```

Comment:
@MimiDumpling "...`this guide will show you how`." kind of implies, "`this guide will show you how to set your own fingerprint`" I think it's clear if they've been reading along. If someone comes to this page and already knows "I just want to see how to set a fingerprint" then I think they would jump down to the "Solutions" section, and wouldn't end up lost.

a phrase like "Change the grouping behavior algorithm" can be spoken so interchangeably with "Set your own fingerprint rule" that it made it hard to decide when and where to write each 😆

Reply:
👍